pkg/controller/perconaservermongodb/psmdb_controller.go

```diff
@@ -262,14 +263,12 @@ func (r *ReconcilePerconaServerMongoDB) Reconcile(request reconcile.Request) (re
 		}
 	}
 
-	for i, replset := range cr.Spec.Replsets {
-		// multiple replica sets is not supported until sharding is
-		// added to the operator
-		if i > 0 {
-			reqLogger.Error(nil, "multiple replica sets is not yet supported, skipping replset %s", replset.Name)
-			continue
-		}
```

Comment:
can we revert this check for the case when Sharding is disabled?

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/439/commits/de73d365fc8da2f252525c6527c404e483a4c60d

Reply:
please return again 😞

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/439/commits/0a7f9f11ccdef730a8864cbbe659bbdc5bd390df